src/app/store/reducers/VehicleTestResultModel.reducers.ts

```diff
@@ -13,15 +13,38 @@ export function VehicleTestResultModelReducers(
       return {
         ...state,
         vehicleTestResultModel: action.payload,
-        error: null // clear error message
+        error: null
```

Comment:
error are controlled globally in the app now. So better to remove the error keys from this file